onnxscript/irbuilder.py

```diff
@@ -468,7 +468,7 @@ def to_function_proto(self) -> onnx.FunctionProto:
             attributes=attribute_names,
             doc_string=self.docstring,
         )
-        if hasattr(onnx.FunctionProto, "attribute_proto"):
+        if hasattr(f, "attribute_proto"):
```

Comment:
Could you add the same note in the Pr description as a comment here? Thanks!

Reply:
I did it

Reply:
Thank you!